src/main/steps/upload-documents/content.test.ts

```diff
@@ -1,47 +1,49 @@
-import { cy, en, form, generateContent } from './content';
+import { cy, en, form, generateContent, getErrors } from './content';
```

Comment:
All file changes in /upload-documents are from DTSSTCI 353 (https://github.com/hmcts/sptribs-dss-update-case-web/pull/35)